lib/scss/components/news-node/_news-node.scss

```diff
@@ -93,8 +108,123 @@
   .news-social-media {
     @include small-paragraph;
 
+    a {
+      color: $su-color-cool-grey-25;
+    }
+
     div {
       display: inline-block;
     }
+
+    i {
+      font-size: 2.5rem;
+
+      // FA is included as a vendor dependency but the class styles are not
+      // included so we have to use the mixins and font declarations here.
+      &::before {
+        @include fa-icon;
+        color: $su-color-black;
+        font-family: 'Font Awesome 5 Brands';
+        font-weight: $su-font-semi-bold; // Need this for the new version of FA. Will go away later.
+        transition: color 0.25s ease-out;
+      }
+    }
+
+    a {
```

Comment:
Multiple of the same selectors are not allowed in a nesting. They should be merged together as the same rule.

@jenbreese just FYI. Resolving in the `default-content` branch.